spec/models/profile_spec.rb

```diff
@@ -264,14 +264,16 @@
     end
 
     it 'does not activate a profile if under fraud review' do
-      profile.update(fraud_review_pending_at: 1.day.ago)
+      # profile.update(fraud_review_pending_at: Time.zone.today - 1.day)
+      profile.fraud_review
       profile.activate
 
       expect(profile).to_not be_active
     end
 
     it 'does not activate a profile if rejected for fraud' do
-      profile.update(fraud_rejection_at: Time.zone.now - 1.day)
```

Comment:
I don't think these date values are significant, only that they make the column not null.